SPECS/expat/expat.spec

```diff
@@ -2,13 +2,14 @@
 Summary:        An XML parser library
 Name:           expat
```

Comment:
This is a toolchain package, so you'll need to update the manifest files under `toolkit/resources/manifests/package/*.txt` to match this version of `expat`.